apps/api/src/bank-transactions-file/helpers/parser.ts

```diff
@@ -6,6 +6,7 @@ import { toMoney } from '../../common/money'
 export const parseString = require('xml2js').parseString
 
 const regexPaymentRef = /\b[A-Z0-9]{4}-[A-Z0-9]{4}-[A-Z0-9]{4}\b/g
+const timeZoneGMTEasternEurope = '+0200'
```

Comment:
Well done, however just thought that we need to adjust for the Daylight Saving Time which is +3 in summer.

As per [Wikipedia for Time in Bulgaria](https://en.wikipedia.org/wiki/Time_in_Bulgaria): 
Daylight saving time moves one hour ahead to [UTC+03:00](https://en.wikipedia.org/wiki/UTC%2B03:00) is observed from the last Sunday in March to the last Sunday in October,[[2]](https://en.wikipedia.org/wiki/Time_in_Bulgaria#cite_note-2) inline with most [EU](https://en.wikipedia.org/wiki/EU) member states.[[3]](https://en.wikipedia.org/wiki/Time_in_Bulgaria#cite_note-3)

Reply:
We can use a standard approach to time conversion 

```tsx
const now = new Date();
const sofiaTime = now.toLocaleString("bg-BG", {timeZone: "Europe/Sofia"});
console.log(sofiaTime); // Output: "12/8/2022, 9:31:03 PM"
```

Reply:
Good catch! Thanks for your suggestions. I decided to use Intl.DateTimeFormat with a slice to get the TimezoneOffset. Unfortunately, I couldn't find an easier and cleaner way to do this